code/modules/antagonists/official/official.dm

```diff
@@ -4,7 +4,6 @@
 	show_in_antagpanel = FALSE
 	var/datum/objective/mission
 	var/datum/team/ert/ert_team
-	can_hijack = HIJACK_PREVENT
```

Comment:
uhh

Reply:
I noticed, don't stress. I'll figure out some way to fix it, but it's approaching midnight so this is a problem for tomorrow me.